apis/clusterresources/v1beta1/awsencryptionkey_types.go

```diff
@@ -39,6 +39,10 @@ type AWSEncryptionKeyStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+//+kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
```

Comment:
Its not age, it is `creationTimestamp`. Age is `timeNow - creationTimestamp`

Reply:
You're right, but there is no way to do arithmetical operations for printing columns. But the way to show the age of a resource is to use `.metadata.creationTimestamp` and type `date`. You may see an example [here](https://github.com/kubernetes/kubectl/issues/903#issuecomment-669244656). K8s does the same thing for all CRDs if there are no additional column printers.

Reply:
Strange a bit, but ok